nomad/local/nomad-agent-conf.nomad

```diff
@@ -42,6 +41,10 @@ client {
     # Turn on consul connect proxy debug logs. Consul connect sidecars now have access logs, etc.
     connect.log_level = "debug"
   }
+  gc_interval              = "30m"
+  gc_disk_usage_threshold  = 99
```

Comment:
We're tuning nomad gc to prevent logs from being collected, right? Might be nice to put a comment pertaining to that above.

Reply:
Yeah, will do

Reply:
done